docker/Dockerfile.tcf-dev

```diff
@@ -160,8 +160,8 @@ RUN cd /project/TrustedComputeFramework/connector_sdk \
 # Environment setup
 
 RUN echo "WEB3_INFURA_PROJECT_ID=9af17e6320414af8a3168cd9fe9d653d" >> /etc/environment \
-    && echo "WALLET_PRIVATE_KEY=4F611197A6E82715F4D2446FE015D1667E9C40A351411F3A7300F71F285D01B4" >> /etc/environment \
-
+ && echo "WALLET_PRIVATE_KEY=4F611197A6E82715F4D2446FE015D1667E9C40A351411F3A7300F71F285D01B4" \
+    >> /etc/environment \
```

Comment:
Both WEB3_INFURA_PROJECT_ID and WALLET_PRIVATE_KEY could be moved to a config file. May be in a separate PR.

Reply:
Both are external to Avalon, so it would be better to not move it to an Avalon-specific file.

But these are separate issues from fixing a syntax error in the Dockerfile.

Reply:
There a config file for block chain connector. So better to move there as a separate PR. @Ram-srini @rranjan3